kyo-kernel/shared/src/main/scala/kyo/kernel/Pending.scala

```diff
@@ -87,12 +85,23 @@ object `<`:
           *   A computation producing the final result
           */
         inline def flatMap[B, S2](inline f: Safepoint ?=> A => B < S2)(
-            using
-            inline frame: Frame,
-            inline flatA: WeakFlat[A],
-            inline flatB: WeakFlat[B]
+            using inline _frame: Frame
         ): B < (S & S2) =
-            map(v => f(v))
```

Comment:
I had to duplicate the body because the compiler was inferring incorrect types and leaking the internal `Kyo` type.